specification/confidentialledger/Microsoft.ManagedCcf/main.tsp

```diff
@@ -0,0 +1,22 @@
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-autorest";
+
+import "./acks.tsp";
+import "./proposals.tsp";
+import "./recovery.tsp";
+import "./service_state.tsp";
+import "./transactions.tsp";
+
+@service({
+  title: "Managed CCF Governance",
+})
+@TypeSpec.Versioning.versioned(Microsoft.ManagedCcf.Versions)
+namespace Microsoft.ManagedCcf;
+
+@doc("Known versions for this service.")
```

Comment:
```suggestion
#suppress "@azure-tools/typespec-azure-core/documentation-required" "https://github.com/Azure/typespec-azure/issues/3107"
```

Version enums should not require docs.  This is a bug in the linter rule we will be fixing, and we are suppressing this error in all specs.